docs/eventing/performance evaluation/nats-jetstream/insvestigation-discrepancy.md

```diff
@@ -0,0 +1,42 @@
+# Why was Events Sent != Events Received in some test runs?
+
```

Comment:
The file name has a typo.
Please rename it to "investigation-discrepancy"

Reply:
done